src/GL/rsp_gl.S

```diff
@@ -648,6 +672,43 @@ GLCmd_PreInitPipe:
     sw t0, %lo(RDPQ_MODE_BLENDER_STEPS) + 0x0
     or modes0, t2
 
+    # Combiner
+    move t5, is_points
+    lw t3, %lo(GL_STATE_TEX_ENV_MODE)
+
+    li t4, GL_REPLACE
+    bne t3, t4, 1f
+    nop
+    ori t5, 0x2
+1:
+
+    beqz active_tex, 1f
+    lw t0, TEXTURE_FLAGS_OFFSET(active_tex)
+    andi t0, TEX_FLAG_COMPLETE
+    srl t0, 1
+    or t5, t0
+1:
+    sll t5, 3
+    lw t0, %lo(COMBINER_TABLE) + 0x0(t5)
+    lw t1, %lo(COMBINER_TABLE) + 0x4(t5)
+    lw t2, %lo(COMBINER_MIPMAPMASK_TABLE) + 0x0(t5)
+    lw t3, %lo(COMBINER_MIPMAPMASK_TABLE) + 0x4(t5)
+
+    # TODO: The following is sort of equivalent to RDPQCmd_ResetMode. Maybe make that callable from ucode?
+
+    sw t0, %lo(RDPQ_COMBINER) + 0x0
+    sw t1, %lo(RDPQ_COMBINER) + 0x4
+    sw t2, %lo(RDPQ_COMBINER_MIPMAPMASK) + 0x0
+    sw t3, %lo(RDPQ_COMBINER_MIPMAPMASK) + 0x4
+
+rdpq_skipcombinerblender:
```

Comment:
I don't think we really have a good naming convention for labels so far, but for now I think we should stay consistent with the rest of the code and use the `gl_` prefix.
```suggestion
gl_skipcombinerblender:
```